### DISCUSSION TITLE
include in TechEmpower Framework Benchmarks

### DISCUSSION THREAD

**Discussion by ewwink** [uNetworking/uWebSockets.js#87] [Q&A] [answered]
Hi,
I saw a lot good web server in there and it will be great if uWebSockets.js is also listed.

website: https://www.techempower.com/benchmarks/
github: https://github.com/TechEmpower/FrameworkBenchmarks

**Comment by ghost** [comment, marked as answer]
No, that is a common request. I do not recommend anything TechEmpower. They are incompetent and cannot properly handle fast servers. No discussion thanks.
